<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E5SslCertificatesDialog</class>
 <widget class="QDialog" name="E5SslCertificatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSL Certificate Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="certificatesWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="serversTab">
      <attribute name="title">
       <string>&amp;Servers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>You have saved certificates identifying these servers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="serversCertificatesTree">
         <column>
          <property name="text">
           <string>Certificate name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Server</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Expiry Date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="serversViewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to view the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;View...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="serversImportButton">
           <property name="toolTip">
            <string>Press to import a certificate</string>
           </property>
           <property name="text">
            <string>&amp;Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="serversExportButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to export the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;Export...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="serversDeleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to delete the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;Delete...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="caTab">
      <attribute name="title">
       <string>Certificate &amp;Authorities</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>You have saved certificates identifying these certification authorities:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="caCertificatesTree">
         <column>
          <property name="text">
           <string>Certificate name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Expiry Date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="caViewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to view the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;View...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="caImportButton">
           <property name="toolTip">
            <string>Press to import a certificate</string>
           </property>
           <property name="text">
            <string>&amp;Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="caExportButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to export the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;Export...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="caDeleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Press to delete the selected certificate</string>
           </property>
           <property name="text">
            <string>&amp;Delete...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>certificatesWidget</tabstop>
  <tabstop>serversCertificatesTree</tabstop>
  <tabstop>serversViewButton</tabstop>
  <tabstop>serversImportButton</tabstop>
  <tabstop>serversExportButton</tabstop>
  <tabstop>serversDeleteButton</tabstop>
  <tabstop>caCertificatesTree</tabstop>
  <tabstop>caViewButton</tabstop>
  <tabstop>caImportButton</tabstop>
  <tabstop>caExportButton</tabstop>
  <tabstop>caDeleteButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>E5SslCertificatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>E5SslCertificatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
